packages/angular/src/stories/diagrams/0-diagrams-start-here.stories.ts

```diff
@@ -1,20 +1,30 @@
 import { storiesOf, moduleMetadata } from '@storybook/angular'
-import { buildElbowPathString, type Coordinates } from '@carbon/charts'
-import { UserModule, WikisModule } from '@carbon/icons-angular'
```

Comment:
The package `@carbon/icons-angular` was recently deprecated per its read me. I switched it over to using `@carbon/icons` directly (ESM).